swagger/v1/component_schemas/ParticipantDeclarationCsvRow.yml

```diff
@@ -54,7 +54,13 @@ properties:
     nullable: false
     example: true
   voided:
-    description: "Indicates whether this declaration has been voided"
+    description: "[Deprecated - use state instead] Indicates whether this declaration has been voided"
```

Comment:
Have you run swaggerise with those changes? I don't think I can see any changes to our api schema file.